packages/nexus-sdk/src/Resource/index.ts

```diff
@@ -124,6 +129,20 @@ const Resource = (
         context: { pollIntervalMs: pollIntervalMs || 1000 },
       });
     },
+    getSource: <T>(
+      orgLabel: string,
+      projectLabel: string,
+      resourceId: string,
+      schemaId?: string,
+      options?: GetResourceSourceOptions,
+    ): Promise<Resource & T> => {
```

Comment:
The `/source` endpoint doesn't return a full resource with all the nexus metadata. It only returns `@type` `@context` and `@id` with whatever the source is.

So I would create a:
```typescript
type ResourceSource = {
  '@context'?: Context;
  '@type'?: string | string[];
  '@id': string; 
}
```
and then:
```suggestion
    ): Promise<ResourceSource & T> => {
```